kiwi/config/functions.sh

```diff
@@ -1,51 +1,29 @@
-#================
-# FILE          : functions.sh
-#----------------
-# PROJECT       : openSUSE Build-Service
-# COPYRIGHT     : (c) 2006 SUSE LINUX Products GmbH, Germany
-#               :
-# AUTHOR        : Marcus Schaefer <ms@suse.de>
-#               :
-# BELONGS TO    : Operating System images
-#               :
-# DESCRIPTION   : This module contains common used functions
-#               : for the config.sh and image.sh scripts
-#               :
-#               :
-# STATUS        : Development
-#----------------
-
-#======================================
-#             IMPORTANT
-#======================================
-# If you change *anything* in this file
-# PLEASE also adapt the documentation
-# in doc/source/working_with_kiwi/shell_scripts.rst
-#======================================
-#             IMPORTANT
-#======================================
-
+#!/bin/bash
+# Copyright (c) 2015 SUSE Linux GmbH.  All rights reserved.
```

Comment:
The "All rights reserved" part is definitely wrong and clashes with the GPL
```suggestion
# Copyright (c) 2021 SUSE Linux GmbH.
```

Reply:
yes will fix it, thanks